README.md

```diff
@@ -0,0 +1,50 @@
+# [motd](#motd)
+
+Configure `/etc/issue` and `/etc/motd` with Ansible.
+
+## [Example Playbook](#example-playbook)
+
+This example is taken from [`molecule/default/converge.yml`](./molecule/default/converge.yml).
+
+```yaml
+---
+- name: Converge
+  hosts: all
+  become: yes  # `no` will also work.
+
+  roles:
+    - role: adfinis.motd
+```
+
+## [Role Variables](#role-variables)
+
+The default values for the variables are set in [`defaults/main.yml`](./defaults/main.yml):
+
+```yaml
+---
+
+# MOTD message (``/etc/motd``).
+motd_message: |
+  -------------------------------------------------
+          This system is managed by Ansible
+  -------------------------------------------------
+
+# Issue message (``/etc/issue`` and ``/etc/issue.net``).
+motd_issue_message: |
+  -------------------------------------------------
+          This system is managed by Ansible
+  -------------------------------------------------
+
+# If to install and use cowsay in the motd messages
+motd_cowsay: false
+```
+
+## [Requirements](#requirements)
+
+- None
+
+## [Compatibility](#compatibility)
+
+This role is only compatible with debian bullseye and bookworm.
+
+The minimum version of Ansible required is 2.12.
```

Comment:
Can you add a footer (see README.rst) in the README.md (of course `s/Adfinis SyGroup/Adfinis/`)

Reply:
Resolved in [44252b8](https://github.com/adfinis/ansible-role-motd/pull/5/commits/44252b85333b2fba23e77d5bc9c83b0644d78a3a)